### PR TITLE
Doc: Fix the typo in doc/rbd/rados-rbd-cmds.rst

### PR DESCRIPTION
**Pull request by FrankYu**
Fix the typo in doc/rbd/rados-rbd-cmds.rst, also add explanation/example
when create image without specifying pool.

Signed-off-by: Frank Yu <flyxiaoyu@gmail.com>

### CODE REVIEW

**Comment by dachary** [comment]
Thanks for the fix :-) Would you mind tell me (and update the commit message) which issue it relates to ?

**Comment by FrankYu** [comment]
dachary,
It's just docs update when I read http://ceph.com/docs/next/rbd/rados-rbd-cmds/.

**Comment by dachary** [comment]
@FrankYu is there an issue for that in http://tracker.ceph.com ? If so you should include Fixes: #number. Also note that the <> around the mail in the commit message must not have white space.  Thanks !

**Comment by FrankYu** [comment]
@dachary There is no issue track in http://tracker.ceph.com. I update the mail info in commit message. Thanks

File: doc/rbd/rados-rbd-cmds.rst [hunk 1]
```diff
@@ -23,12 +23,17 @@ the  following::
...
 	rbd create {image-name} --size {megabytes} --pool {pool-name}
 	
-For example, to create a 1GB image named ``foo`` that stores information in a 
+For example, to create a 1GB image named ``bar`` that stores information in a 
 pool named ``swimmingpool``, execute the following::
 
-	rbd create foo --size 1024
 	rbd create bar --size 1024 --pool swimmingpool
 
+If you don't specify pool when create a image, the image will be stored in the
```
Comment (dachary) [inline doc/rbd/rados-rbd-cmds.rst]: s/create a image, the image/creating an image, it/

File: doc/rbd/rados-rbd-cmds.rst [hunk 2]
```diff
@@ -23,12 +23,17 @@ the  following::
...
+default pool ``rbd``. For example, create a 1GB image named ``foo`` that stores
```
Comment (dachary) [inline doc/rbd/rados-rbd-cmds.rst]: * s/create a 1GB/to create a 1GB/
* s/that stores information/stored/

**Comment by FrankYu** [comment]
@dachary 
Sorry for my mistake to close this PR. I open new one #3433
And, thank you very much, dachary

**Comment by dachary** [comment]
@FrankYu it's fine :-) Note for the future that you can re-open a pull request that you closed by accident, you do not need to open a new one.

**Comment by FrankYu** [comment]
@dachary , yep. I got it. Thanks :)

**Pull request closed without merging**